### PR TITLE
feat(playlist-carousel): updated playlist library to use carousel on desktop

### PR DESCRIPTION
**Pull request by exscruzme**

### CODE REVIEW

File: src/lib/components/PlaylistCarousel.svelte [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    font-style: italic;
+    color: #9f9f9f;
+  }
+
+  @media (max-width: 768px) {
+    .item {
+      margin: 0.5rem;
+      padding: 0.5rem;
+    }
+    .section {
```
Comment (exscruzme) [inline src/lib/components/PlaylistCarousel.svelte:131]: I figured if the user is looking at the library on table or mobile then scrolling would be better than the buttons

**Comment by admoya** [comment]
Looks very nice! The scrolling is working very well and animations looks slick. This probably is better than pages. I do think the buttons kind of don't match the style of the rest of the page though, and maybe we should have 4 buttons (with just arrows, no words), `All the way back`, `back`, `forward`, `all the way forward`.

Also, the images individually are not that big but when you throttle your internet a little it still takes forever to load (though the page doesn't shift around so that is good). But maybe if we use the sveltekit built-in `enhanced:img` feature it would scale them down to the smallest size possible, worth a try and then comparing the loading times: https://kit.svelte.dev/docs/images#sveltejs-enhanced-img

**Comment by admoya** [comment]
Also the fact that it is still a normal scroll in mobile is very cool, works well. I think we should make the carousel generic and put the decks in there too, at least on mobile, because it is a bit annoying to scroll past all those. And maybe we could hide the scrollbar on the bottom cause it gets so small as to be nearly useless. So maybe hiding that and then having some kind of indication that you can swipe

**Comment by exscruzme** [comment]
I should say also that the carousel is a modified version of a sandbox example i found! 

https://codesandbox.io/p/sandbox/simple-svelte-carousel-8t5rr?file=/Carousel.svelte

**Comment by exscruzme** [comment]
Changes included in https://github.com/CSMA-Technology/blend-product-site/pull/128

**Pull request closed without merging**